### DISCUSSION TITLE
Aggregate annotations from multiple items in the side panel

### DISCUSSION THREAD

**Discussion by kjanel** [aidenlx/obsidian-zotero#519] [Ideas]
I would like to be able to view annotations coming from multiple documents in the side panel (currently we need to select one specific Zotero item.

The annotations to display could be specified using filters (relating to annotations or parent items metada).

Alternatively or as a complement, it would be very useful to have an option to export each annotation as a single note (in addition to the current workflow). This way, these annotations would be standalone notes with their properties in Obsidian, and we could query these annotations using plugins like dataview or even display them on obsidian canvas (since it's not so easy in obsidian to manipulate these annotations if they are "blocks"

**Comment by aidenlx** [comment]
Note: the "export each annotation as a standalone note" idea in the second paragraph is the same request as #313 — following it over there. Keeping this issue scoped to multi-item aggregation in the side panel.

**Comment by aidenlx** [comment]
Note: the per-annotation-note aspect of this request is tracked in #313. Keeping this issue open for the distinct multi-item annotation aggregation feature (viewing annotations from multiple items in the side panel).
